src/sagas/index.js

```diff
@@ -0,0 +1,7 @@
+import { all } from 'redux-saga/effects';
+import giphySaga, { loadTrendingSaga } from './giphyRequests.js';
+
+export default function* rootSaga() {
+	//console.log('Saga loaded');
+	yield all([loadTrendingSaga(), giphySaga()]);
```

Comment:
You should dispatch an action to call `loadTrendingSaga()`. I believe you are trying to load on app start (correct me if im wrong). So probably inside a `componentDidMount` dispatch the loadTrending action. This way you avoid also exporting the saga.